### PR TITLE
support for aes fpe key creation & encryption-decryption

### PR DESCRIPTION
**Pull request by akash-fortanix**

### CODE REVIEW

File: src/main/java/com/fortanix/sdkms/performance/helper/AESHelper.java [hunk 1]
```diff
@@ -40,10 +40,17 @@
     @Override
     public EncryptRequestEx createEncryptRequest(SobjectDescriptor key, String plain) {
         ByteArrayOutputStream input = new ByteArrayOutputStream();
+        if(mode == CryptMode.FPE) {
+            plain = "36088650107272";
```
Comment (manasag) [inline src/main/java/com/fortanix/sdkms/performance/helper/AESHelper.java]: Why are we overriding the plain value? This means user won't be able to set a custom value.

File: src/main/java/com/fortanix/sdkms/performance/sampler/AbstractEncryptionAndDecryptionSampler.java [hunk 2]
```diff
@@ -54,6 +55,10 @@ public void setupTest(JavaSamplerContext context) {
         super.setupTest(context);
         this.securityObjectsApi = new SecurityObjectsApi(this.apiClient);
         SobjectRequest sobjectRequest = new SobjectRequest().name(UUID.randomUUID().toString()).objType(objectType).keySize(keySize);
+        if(CryptMode.fromValue(mode) == CryptMode.FPE) {
+            FpeOptions fpeOptions = new FpeOptions().radix(10).preserve(Arrays.asList(0,1,2,3,-1,-2,-3,-4)).luhnCheck(true).name("Credit Card");
```
Comment (manasag) [inline src/main/java/com/fortanix/sdkms/performance/sampler/AbstractEncryptionAndDecryptionSampler.java]: Please add a comment mentioning that currently we are supporting only credit card numbers for FPE encryption.

File: src/main/java/com/fortanix/sdkms/performance/helper/AESHelper.java (hunk 1 above)
Comment (akash-fortanix) [reply src/main/java/com/fortanix/sdkms/performance/helper/AESHelper.java]: Taken care of.

File: src/main/java/com/fortanix/sdkms/performance/sampler/AbstractEncryptionAndDecryptionSampler.java (hunk 2 above)
Comment (akash-fortanix) [reply src/main/java/com/fortanix/sdkms/performance/sampler/AbstractEncryptionAndDecryptionSampler.java]: Taken care of.

**Pull request merged**